.cirrus/tasks.yml

```diff
@@ -174,13 +169,9 @@ chart_fixture_test_task:
     memory: 1Gb
   <<: *CLONE_SCRIPT_TEMPLATE
   script:
-    - helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
-    - helm repo add bitnami-pre2022 https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
-    - cd ${CIRRUS_WORKING_DIR}/charts/sonarqube
-    - helm dependency build
-    - cd ${CIRRUS_WORKING_DIR}/charts/sonarqube-dce
-    - helm dependency build
-    - cd ../..
+    - cd "${CIRRUS_WORKING_DIR}" && ./.cirrus/build_chart_dependencies.sh charts/sonarqube
+    - cd "${CIRRUS_WORKING_DIR}" && ./.cirrus/build_chart_dependencies.sh charts/sonarqube-dce
+    - cd ${CIRRUS_WORKING_DIR}
```

Comment:
Same here all CD could be removed I guess

Reply:
Nice catch! Thanks